webpack.config.js

```diff
@@ -60,7 +60,7 @@ module.exports = {
 			{
 			path: path.join( __dirname, 'dist' ),
 			filename: '[name]-' + process.env.npm_package_version + '.js',
-			chunkFilename: 'chunks/[name].[chunkhash].min.js',
```

Comment:
@frosso @Ferdev @c-shultz @bborman22 I want to double check, is there a reason why we should use `'chunks/[id].[chunkhash].min.js'` or is `chunks/[chunkhash].min.js` okay?

Reply:
I'm not sure - what would be the advantage of adding the `id`? 👀 
I think that just using the `chunkhash` would be fine for now. It's still going to change between builds/changes, so that the browser's cache is busted 👍